docs/ir/ir-breaking-changes.rst

```diff
@@ -4,3 +4,32 @@ Solidity IR-based Codegen Changes
 
 This section highlights the main differences between the old and the IR-based codegen,
 along with the reasoning behind the changes and how to update affected code.
+
+Semantic Only Changes
+=====================
+
+This section lists the changes that are semantic-only, thus potentially
+hiding new and different behavior in existing code.
+
+ * If storage struct occupies less than 32 bytes in storage and we want to delete it, full slot will be set to zero.
```

Comment:
```suggestion
 * When storage structs are deleted, every storage slot that contains a member of the struct is set to zero entirely. Formally, padding space was left untouched.
```

Reply:
Note that this is neither restricted to structs that are shorter than 32 bytes, nor to the end of the struct.
I.e. ``struct { uint8 x; uint256 y; }`` is affected in the padding space of ``x``.